designer/server/src/test/scala/pl/touk/nussknacker/ui/component/ComponentsUsageHelperTest.scala

```diff
@@ -131,11 +139,49 @@ class ComponentsUsageHelperTest extends AnyFunSuite with Matchers with TableDriv
 
   private val processDetailsWithFragment = displayableToProcess(TestProcessUtil.toDisplayable(processWithFragment))
 
-  private val processingTypeAndComponentInfoToComponentId: (ProcessingType, ComponentInfo) => ComponentId = {
-    case (_, ComponentInfo(ComponentType.CustomComponent, `otherExistingStreamTransformer`)) =>
-      ComponentId(overriddenOtherExistingStreamTransformer)
-    case (processingType, info) =>
-      ComponentId.default(processingType, info)
+  private def nonFragmentComponents(componentInfoToId: ComponentInfo => ComponentId) = {
+    val modelDefinition = ModelDefinitionBuilder
```

Comment:
From what place did you duplicate this code? I won't believe that you wrote it from scratch :) Can you deduplicate this code?

Reply:
i'ts not really duplication, i've used ModelDefinitionBuilder analogously to ProcessTestData/ProcessValidatorSpec but included only the components needed by the test